### ISSUE TITLE
WHMCS: False Positive with 942350 and 932130 When Saving Knowledgebase Articles

### ISSUE THREAD

**Issue by linux4me** [coreruleset/coreruleset#1850]
### Description
I got false positives with rules 942350 and 932130 when attempting to save Support Knowledgebase article edits in the admin of WHMCS v. 7.10.2. 

I'm not sure it's relevant, but this particular installation has the temporary directory as well as local storage directories located outside the public_html directory, as recommended by WHMCS.

### Audit Logs / Triggered Rule Numbers

See attached.

### Your Environment

* CRS version (e.g., v3.2.0): 3.2.0
* Paranoia level setting: 1
* ModSecurity version (e.g., 2.9.3): 2.9.3
* Web Server and version (e.g., apache 2.4.41): 2.4.41
* Operating System and version: CentOS 7 Linux 3.10.0-1127.8.2.el7.x86_64

### Confirmation

[ X] I have removed any personal data (email addresses, IP addresses,
    passwords, domain names) from any logs posted.

**Comment by linux4me** [comment]
[WHMCS_KB_issues.txt](https://github.com/coreruleset/coreruleset/files/4933477/WHMCS_KB_issues.txt)

**Comment by lifeforms** [comment]
Hi @linux4me , thanks for reporting this problem!

It seems that WHCMS has a function to write articles, which seems like a sort of wiki.
Unfortunately, these free text fields have a tendency to contain code which looks suspicious to a WAF.

We don't have a WHCMS exclusion profile, sadly, so that means you might hit into false positives.

Can you add custom configuration to your webserver? If so, you could try adding the following rule:

```
SecRule REQUEST_FILENAME "@endsWith /whmcs/admin/supportkb.php" \
	"id:123456,phase:1,t:none,nolog,pass,chain"
		SecRule ARGS:action "@streq save" "t:none,\
			ctl:ruleRemoveTargetByTag=OWASP_CRS;ARGS:article"
```

What this does, is disable the WAF checking the article field. This should hopefully solve your problems!

If not, I might have made a mistake, so feel free to re-open!
